### DISCUSSION TITLE
Dockge not updating local host ip for containers

### DISCUSSION THREAD

**Discussion by myths420** [louislam/dockge#598] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

been playing with this testing out before moving to main server. ive had to change ip of test server. dockge updates to the new ip and runs fine but wont update any of the containers with new local host ip. how can i get them working again? even setting ip in compose isnt updating containers local host ip.

### 🐻 Dockge Version

1.4.2

### 💻 Operating System and Arch

truenas scale

### 🌐 Browser

chrome

**Comment by trythatagain** [comment]
Need more info, not sure how this could happen. How are you accessing the new containers? Are you spinning up brand new containers including appdata folders?

**Reply by myths420** [reply]
ah after endlessly trying i got it to work. most things ive used that when you change the host ip it will update containers this one has a single setting in profile hidden i had to change ip there lol.
